### PR TITLE
Update a task

### PR DESCRIPTION
**Pull request by MikAleinik**
#### Title of Pull Request

Update a task after the task is completed

#### 🤔 This is a ...

- [ ] 🌟 New task
- [ ] 🌐 New module
- [x] ⚙️ Update to an existing task
- [ ] 🔧 Update to an existing module
- [ ] 🔗 Update or addition of external resources or links
- [ ] 🐛 Fix in a task or related content
- [ ] 🛠 Fix in a module or related content
- [ ] ✏️ Fixed a typo or grammatical error
- [ ] 🔗 Fixed a broken link
- [ ] ❓ Other (specify: **\*\*\*\***\_\_\_\_**\*\*\*\***)

#### Description

- added a linter rule to exclude the use of inline settings
- added test for checking commented code

#### Checklist

- [x] ✅ I have performed a self-review of my own code.
- [x] 📝 I have commented my code, particularly in hard-to-understand areas.
- [x] 🔧 I have made corresponding changes to the documentation (if applicable).
- [x] 🚫 My changes generate no new warnings or errors.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -129,3 +129,5 @@ dist
 .yarn/install-state.gz
 .pnp.*
 /src/_numbers-tasks.js
+/package-lock.json
+/package-lock.json
```
Comment (gentoosiast) [inline .gitignore:133]: Why you have two identical records?

File: test/numbers-tests.js [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
+const forbidden = require('../extensions/forbidden');
 
 describe('core-js-numbers', () => {
   it.optional('getRectangleArea should return a square of rectangle', () => {
     assert.equal(tasks.getRectangleArea(5, 10), 50);
     assert.equal(tasks.getRectangleArea(5, 5), 25);
+    assert.equal(
+      forbidden.isCommented(tasks.getRectangleArea),
+      false,
+      `Be sure to remove comments from the final solution`
```
Comment (gentoosiast) [inline test/numbers-tests.js:13]: Why you use backticks if there are no interpolation of variables inside them?

File: test/numbers-tests.js [hunk 3]
```diff
@@ -275,19 +383,29 @@ describe('core-js-numbers', () => {
     () => {
       assert.equal(tasks.isInteger(5), true, 'value 5 is an integer');
       assert.equal(tasks.isInteger(5.1), false, 'value 5.1 is not an integer');
-      assert.equal(tasks.isInteger('5'), false, 'value 3/2 is not an integer');
+      assert.equal(tasks.isInteger('5'), false, 'value "5" is not an integer');
```
Comment (gentoosiast) [inline test/numbers-tests.js:386]: Please make messages consistent. If you use double quotes around value in the last message, add such quotes to other messages.

Comment (MikAleinik) [reply test/numbers-tests.js:386]: Thank you! Corrected.

File: test/numbers-tests.js (hunk 2 above)
Comment (MikAleinik) [reply test/numbers-tests.js]: Thank you! Corrected.

File: .gitignore (hunk 1 above)
Comment (MikAleinik) [reply .gitignore]: Thank you! Corrected.

**Review by stardustmeg — APPROVED** [review]

**Pull request merged**